javascriptsource/datawidgets/actions/Export_To_Excel.js

```diff
@@ -0,0 +1,78 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+import "mx-global";
+import { Big } from "big.js";
+import { utils, writeFileXLSX } from './xlsx-export-tools.js';
+
+// BEGIN EXTRA CODE
+// END EXTRA CODE
+
+/**
+ * @param {string} datagridName
+ * @param {string} fileName
+ * @param {string} sheetName
+ * @param {boolean} includeColumnHeaders
+ * @param {Big} chunkSize - The number of items fetched and exported per request.
+ * @returns {Promise.<boolean>}
+ */
+export async function Export_To_Excel(datagridName, fileName, sheetName, includeColumnHeaders, chunkSize) {
+	// BEGIN USER CODE
+    if (!fileName || !datagridName || !sheetName) {
+        return false;
+    }
+
+    return new Promise((resolve, reject) => {
+        const DATAGRID_DATA_EXPORT = "com.mendix.widgets.web.datagrid.export";

```

Comment:
shall we move the const to the top instead? or just put the string inside the window[""] directly. otherwise, having a separate const here is not needed.